src/tx/script/op-codes/op-code.ts

```
@@ -0,0 +1,1135 @@
+      case OpCodes.OpRot:
+        vm.dstack.rotItems(1);
+        break;
+
+      case OpCodes.OpSwap:
+        vm.dstack.swapItems(1);
+        break;
+
+      case OpCodes.OpTuck:
+        if (vm.dstack.length < 2) TxScriptError.throwInvalidStackOperation(2, vm.dstack.length);
+
+        const item = vm.dstack[vm.dstack.length - 1];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)